notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    def prep(x):
</pre><p>W sumie, sam nie korzystam z tego za często, ale mamy takie magiczne metody w Pandasie jak <code>cut</code>  i <code>qcut</code>  - dobrze wiedzieć bo robią dokładnie to o co Ci chodzi.</p><p>https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.cut.html?highlight=cut#pandas.cut</p><p>https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.qcut.html#pandas.qcut</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>